src/csharp/Grpc.Core/ServerServiceDefinition.cs

```diff
@@ -63,28 +63,24 @@ private ServerServiceDefinition(Dictionary<string, IServerCallHandler> callHandl
         /// <summary>
         /// Creates a new builder object for <c>ServerServiceDefinition</c>.
         /// </summary>
-        /// <param name="serviceName">The service name.</param>
         /// <returns>The builder object.</returns>
-        public static Builder CreateBuilder(string serviceName)
+        public static Builder CreateBuilder()
         {
-            return new Builder(serviceName);
+            return new Builder();
         }
 
         /// <summary>
         /// Builder class for <see cref="ServerServiceDefinition"/>.
         /// </summary>
         public class Builder
         {
-            readonly string serviceName;
             readonly Dictionary<string, IServerCallHandler> callHandlers = new Dictionary<string, IServerCallHandler>();
 
             /// <summary>
             /// Creates a new instance of builder.
             /// </summary>
-            /// <param name="serviceName">The service name.</param>
-            public Builder(string serviceName)
+            public Builder()
```

Comment:
I plan to introduce an automatic code formatter at some point, and after that I'll fix all of these at once.